packages/@react-aria/overlays/src/calculatePosition.ts

```diff
@@ -181,7 +181,7 @@ function getDelta(
   // Note that these values are with respect to the visual viewport (aka 0,0 is the top left of the viewport)
   let boundaryStartEdge = boundaryDimensions.scroll[AXIS[axis]] + padding;
   let boundaryEndEdge = boundarySize + boundaryDimensions.scroll[AXIS[axis]] - padding;
-  let startEdgeOffset = offset - containerScroll + containerOffsetWithBoundary[axis] - boundaryDimensions[AXIS[axis]];
+  let startEdgeOffset = offset - containerScroll + containerOffsetWithBoundary[axis];
```

Comment:
lol positioning code changes again, I'm going to have to take some time to do the math/diagramming again just to make sure this is what we want. Mind running chromatic just in case?